packages/replay/src/main.ts

```diff
@@ -229,6 +236,15 @@ function updateStatus(recording: RecordingEntry, status: RecordingEntry["status"
   recording.status = status;
 }
 
+function filterRecordings(recordings: RecordingEntry[], filter?: string) {
+  if (filter) {
+    const exp = jsonata(`$filter($, ${filter})[]`);
```

Comment:
What was the reason for using jsonata here (instead of the regular array filter)??

Reply:
We need something that can be fed in via the CLI and jsonata enables that more easily than writing our own that translates to an array filter.